operator/option/config.go

```diff
@@ -446,15 +447,15 @@ func (c *OperatorConfig) Populate() {
 		c.IPAMSubnetsIDs = m
 	}
 
-	if m := viper.GetStringMapString(IPAMSubnetsTags); len(m) != 0 {
+	if m := command.GetStringMapString(viper.GetViper(), IPAMSubnetsTags); len(m) != 0 {
```

Comment:
I think we should use `GetStringMapStringE` here, now that we have it, and fail with `log.Fatalf` for any parsing errors. The underlying issue this PR is intending to fix wasn't discovered for almost two years because viper silently swallowed the errors.